README.md

```diff
@@ -9,7 +9,7 @@
 <p align=center>
 <a href="https://iina.io">Website</a> ·
 <a href="https://github.com/iina/iina/releases">Releases</a> ·
-<a href="https://t.me/joinchat/AAAAAEBemW7dU8X7IHShwQ">Telegram Group</a>
+<a href="https://t.me/joinchat/IINAUsers">Telegram Group</a>
```

Comment:
The link should be https://t.me/IINAUsers

Reply:
Thanks! Fixed.